src/site/content/en/fast/chrome-ux-report-pagespeed-insights/index.md

```diff
@@ -35,85 +35,96 @@ controlled test in the lab.
+- **FID** measures the interactivity of the page, from the user's first
+  interaction to the time the page responds to it.
+- **LCP** measures the time until the page displays what is likely its main
+  content, like a hero image or heading.
+- **CLS** measures the degree of layout instability on the page, due to shifts
+  like asynchronously loaded content being injected.
+
+Metric | Good | Needs Improvement | Poor
 -- | -- | -- | --
-FCP | 0–1000ms | 1000ms–2500ms | 2500ms+
-FID | 0–50ms | 50–250ms | 250ms+
+FCP | 0–1000ms | 1000ms–3000ms | 3000ms+
+FID | 0–100ms | 100–300ms | 300ms+
+LCP | 0–2500ms | 2500–4000ms | 4000ms+
+CLS | 0.00-0.10 | 0.10–0.25 | 0.25+
 
-This table describes how FCP and FID values are categorized as either fast,
-average, or slow.
+This table describes how values for these metrics are categorized as either
+good, needing improvement, or poor.
 
 There are three ways that the user experience is displayed in PSI:
 
-- a label summarizing the page as fast, average, or slow
-- a percentile measured in seconds or milliseconds
-- a distribution representing the percent of fast, average, and slow experiences
-
-In the screenshot above, the page has a speed label of "average". To achieve a
-"fast" label, both FCP and FID must be categorized as fast. A page is labelled
-as "slow" if either metric is categorized as slow. All other cases are labelled
-as "average".
-
-A percentile is shown for each metric. For FCP, the 90th percentile is used.
-For FID, it's the 95th percentile. A percentile is a measure used in statistics
-indicating the value below which a given percentage of observations in a group
-of observations fall. For example, the 20th percentile is the value below which
-20% of the observations may be found. Using the screenshot above as an example,
-90% of FCP experiences on this page are 2 seconds or less. Similarly, 95% of FID
-are 149ms or less. These values are also color-coded according to the speed
-label rules, where fast values are green, average values are orange, and slow
+- a label summarizing the page as passing or not passing the Core Web Vitals 
+  assessment
+- percentiles measured in seconds or milliseconds (CLS is unitless)
+- a distribution representing the percent of good, needing improvement, and
+  poor experiences
+
+In the screenshot above, the page is labelled as "passing" the Core Web Vitals
+assessment. To pass, the percentile must be categorized as "good".
+Otherwise, the page is labelled as "not passing".
+
+The percentiles shown for all metrics correspond to the 75th percentile. In
+statistics, a percentile is a measure that indicates the value below which a
+given percentage of samples fall. For example, the screenshot above shows that
+FID's 75th percentile is 15ms, meaning that 75% of FID experiences are faster
+than 15ms. These values are color-coded according to the threshold table above
+where good values are green, values needing improvement are orange, and poor
 values are red.
```

Comment:
It might be good to consistently wrap the 3 evaluation keywords consistently in quotes. E.g. "good" and "needs improvement", and "poor". And to do this consistently throughout all updated docs.